Makefile

```diff
@@ -49,6 +50,8 @@ lib/rescript.exe: $(CLI_FILES)
 
 build-native: lib/refmt.exe lib/rescript.exe depend
 
+build-ppx: src/reactjs_jsx_ppx.cmx
```

Comment:
why does the ppx need a separate `build-ppx` rule? If it's a part  "API_FILES", is this sufficient?